libs/testing/src/user.session.ts

```diff
@@ -84,7 +77,6 @@ export class UserSession {
     options: {
       noOrganization?: boolean;
       noEnvironment?: boolean;
-      noIntegrations?: boolean;
```

Comment:
I removed this. It was only used in 2 tests. In one of the tests didn't make sense at all to not have default integrations and the other one was doing something that could be checked with the existing test data.